va/va.h

```diff
@@ -980,6 +991,31 @@ typedef union _VAConfigAttribValEncJPEG {
 #define VA_ENC_INTRA_REFRESH_MULTI_REF                  0x00040000
 
 /**@}*/
+/** @name Attribute values for VAConfigAttribHistogram channel. */
+/**@{*/
+typedef enum{
+    VAHistogramChannelNone = 0,
+    VAHistogramChannelY,
+    VAHistogramChannelU,
+    VAHistogramChannelV,
+    VAHistogramChannelR,
+    VAHistogramChannelG,
+    VAHistogramChannelB,
+    VAHistogramChannelA
+}VAHistogramChannelType;
+
+/** \brief Attribute value for VAConfigAttribHistogram */
+typedef union _VAConfigAttribHistogram{
+    struct {
+        /** \brief bin count for given channel. */
+        uint32_t bin_count : 16;
+        /** \brief driver support channel */
+        uint32_t channel   : 8;
```

Comment:
does it support multiple channels?

Reply:
yes, for the caps report, will return several VAConfigAttribHistograms , also application can send multiple VAEncMiscParameterHistorgram